### ISSUE TITLE
Signatures not showing

### ISSUE THREAD

**Issue by girls-whocode** [barisusakli/nodebb-theme-peace#3]
I have been using this template and really like the layout and design. Although it will not show the signatures. I looked at NodeBB's API and it is showing there. I started to look for it in the code, although I have not had luck. Is this a bug?

**Comment by girls-whocode** [comment]
Ok, I had a little time to dig into the code. This part is missing from the /nodebb/node_modules/nodebb-theme-peace/templates/partials/topic/post.tpl on line 95

```
{{{ if posts.user.signature }}}
	<div component="post/signature" data-uid="{posts.user.uid}" class="post-signature">{posts.user.signature}</div>
{{{ end }}}
```

I did just edit yours directly from the folder. I am hoping this will get put in future versions. If not I will clone yours and build my own. Thanks for listening.

**Comment by barisusakli** [comment]
published 2.2.4
